src/utils/checkRequired.js

```diff
@@ -0,0 +1,7 @@
+const checkRequired = (data, message) => {
+  if (!data) {
+    return `${message} is required`;
+  }
+};
```

Comment:
Consider returning 'null' or 'undefined' explicitly when the data is present. This will make it clear that the function returns a string only when the data is missing.